### ISSUE TITLE
multilingual database

### ISSUE THREAD

**Issue by edsonee** [arkypita/LaserGRBL#1719]
Is there any way to create the database in multilanguages?

![atomstack A5 (20w)](https://user-images.githubusercontent.com/83145998/149564576-4bf5899f-1ab9-4e2c-8963-817ec5170765.png)

**Comment by arkypita** [comment]
No, not jet. Moreover the translations are done by volunteers and, often, once they are done they remain out of date.
If you want you can edit the DB from the menu "grbl" "material database"
